concepts/security/README.md

```diff
@@ -1,5 +1,14 @@
 # Security
 
+End-to-end encryption for communication between microservices is generally a best practice, but it's also hard to set up. A common way to secure microservice inter-communication is to use [mutual authentication TLS](https://en.wikipedia.org/wiki/Mutual_authentication) or mTLS. `mTLS` offers a few key features for network traffic inside your app:
```

Comment:
This intro is fine to add, however this security topic is MORE than just mTLS security, so really it should cover brief sentence on component, binding security features.